CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Medrunner API Client Contributing Guide
+
+We're really excited that you are interested in contributing to the Medrunner API Client. Before submitting your contribution, please make sure to take a moment and read through the [Code of Conduct](https://github.com/medrunner-services/api-client-ts/blob/main/CODE_OF_CONDUCT.md).
+
+## Contributing by creating issues
+
+The simplest and easiest way to contribute to the project is to create an issue. You can report bugs, request features, or suggest improvements. This way we know what is important for you and we can work on it faster!
+
+When creating an issue, be sure to provide the maximum amount of information and context so that we can understand the problem or the suggestion. If you are reporting a bug, please provide the environnement where the bug occurred and a way to reproduce it.
+
+## Contributing by creating pull requests
+
+If you want to directly contribute to the project by adding or fixing some code, you can create a pull request with your desired changes. Here are some guidelines to follow:
+
+- Fork the repository into your own account.
+
+- Checkout a topic branch from `main` with a name like `feature/adding-types` or `bug/docs-images-error`.
+
+- Make sure to run `npm run lint` and `npm run prettier` before committing your changes. You can even enable GitHub actions in your forked repository to run these checks automatically.
```

Comment:
Should this be lint:fix and prettier:fix?

Reply:
Yeah true

Reply:
Changed !